docs/source/tutorial/gpu.rst

```diff
@@ -107,29 +107,30 @@ It is equivalent to the following code using CuPy:
    If user uses only one device, these device switching is not needed.
    :func:`chainer.cuda.to_cpu` and :func:`chainer.cuda.to_gpu` functions automatically switch the current device correctly.
 
-Chainer also provides a convenient function :func:`chainer.cuda.get_device` to select a device.
-It accepts an integer, CuPy array, NumPy array, or None (indicating the current device), and returns an appropriate device object.
-If the argument is a NumPy array, then *a dummy device object* is returned.
+Chainer also provides convenient functions :func:`chainer.cuda.get_device_from_id` and :func:`chainer.cuda.get_device_from_array` to select a device.
+The former one accepts an integer indicating the target device and returns an appropriate device object.
+The latter one accepts a :class:`cuda.ndarray` object and returns the corresponding device object.
+In this latter function, if the argument is a NumPy array, then *a dummy device object* is returned.
```

Comment:
* I think the word "convenient" is no longer applicable. The word implies doing something comlicated for the user, but its functionalities are now simplified.
* Each appearance of the word "one" seems redundant for me. "The former[latter] accepts" feels like more natural.